src/xunit.netcore.extensions/TargetFrameworkMonikers.cs

```diff
@@ -3,33 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using Xunit.Sdk;
 
 namespace Xunit
 {
     [Flags]
     public enum TargetFrameworkMonikers
     {
-        Net45 = 0x1,
```

Comment:
While I agree we only plan to use the versionless ones at this point in time does keeping the versioned ones hurt anything? Just in case we change our mind or do want to add a few specific versioned tests that aren't latest?